packages/xrpl-crypto/LICENSE

```diff
@@ -1,6 +1,6 @@
 ISC License
 
-Copyright (c) 2012-2015 Ripple Labs Inc.
+Copyright (c) 2012-2023 The XRPL developers
```

Comment:
I guess this particular package only really starts at 2023?

Reply:
That seems accurate to me